### ISSUE TITLE
incompatibility with lithium's latest neoforge version for 1.21.1

### ISSUE THREAD

**Issue by VaporeonScripts** [malte0811/FerriteCore#174]
https://mclo.gs/2MyYH4B

**Comment by VaporeonScripts** [comment]
https://github.com/CaffeineMC/lithium-fabric/issues/569#issuecomment-2417394847

**Comment by malte0811** [comment]
See #173: This is fixed in the dev build from this GH action run: https://github.com/malte0811/FerriteCore/actions/runs/11371242578 (the previous run did not upload artifacts due to a build script bug). Given that Lithium on Neo is only available from GH action builds too, I do not consider it important enough to create a new release for it. 1.21.2 seems to be close to a release, the "fix" will be included in the FC build for that.

**Comment by VaporeonScripts** [comment]
Yeah but I won't use 1.21.2 for my modpack? It should be back ported tho :C

**Comment by malte0811** [comment]
If they actually release Lithium on Neo on 21.1 I'll look into making a new release. But the only statement I got from them so far was along the lines of "we don't know yet, maybe we'll do backports". Also, you can easily apply the "fix" yourself in a pack since it's just an option in the Lithium config.

**Comment by VaporeonScripts** [comment]
bro they just did :sob:

**Comment by VaporeonScripts** [comment]
https://www.curseforge.com/minecraft/mc-mods/lithium/files/5817196

**Comment by VaporeonScripts** [comment]
> Also, you can easily apply the "fix" yourself in a pack since it's just an option in the Lithium config.

uh, where? also i cant even launch minecraft, so how can i edit that, i think u mean from config folder?

**Comment by VaporeonScripts** [comment]
![image](https://github.com/user-attachments/assets/6498a945-a4ba-40d3-910a-86f74af5b43f)
no config for lithium lol

**Comment by malte0811** [comment]
I was not aware that they had released one 4 days after telling me that they weren't sure yet whether they would. So I assumed that you were one of the people testing their GH builds. I will try to get a build out tomorrow after I get back from work. You can ask the lithium team about instructions for the config, but GH issues are not for "chat-style" one-message-per-5-seconds conversations.

**Comment by malte0811** [comment]
I did not get around to it the last two days (quite busy at work), but I've put the build on CF/MR now. CF still needs to be approved, MR should already be visible.

**Comment by VaporeonScripts** [comment]
alright
